host_core/lib/host_core/vhost/config_plan.ex

```diff
@@ -30,7 +30,7 @@ defmodule HostCore.ConfigPlan do
           {:rpc_timeout_ms, "WASMCLOUD_RPC_TIMEOUT_MS",
            required: false, map: &String.to_integer/1},
           {:rpc_jwt, "WASMCLOUD_RPC_JWT", required: false},
-          {:rpc_tls, "WASMCLOUD_RPC_TLS", required: false, map: &String.to_integer/1},
+          {:rpc_tls, "WASMCLOUD_RPC_TLS", required: false, map: &string_to_bool/1},
```

Comment:
For these `string_to_bool` that aren't required, do they need a default specified?